mmv1/third_party/terraform/.teamcity/components/generated/project.erb

```diff
@@ -6,54 +6,96 @@
 
 // this file is auto-generated with mmv1, any changes made here will be overwritten
 
-import jetbrains.buildServer.configs.kotlin.BuildType
-import jetbrains.buildServer.configs.kotlin.Project
-import jetbrains.buildServer.configs.kotlin.AbsoluteId
+import jetbrains.buildServer.configs.kotlin.*
 
 const val providerName = "google<%= "-" + version unless version == 'ga' -%>"
 
 // Google<%= version.capitalize unless version == 'ga' -%> returns an instance of Project,
 // which has multiple build configurations defined within it.
 // See https://teamcity.jetbrains.com/app/dsl-documentation/root/project/index.html
 fun Google<%= version.capitalize unless version == 'ga' -%>(environment: String, manualVcsRoot: AbsoluteId, branchRef: String, configuration: ClientConfiguration) : Project {
+
+    // Create build configs for each package defined in packages.kt
+    val packageConfigs = buildConfigurationsForPackages(packages, providerName, environment, manualVcsRoot, configuration)
+
+    // Create build configs for each service package defined in services.kt
+    val servicePackageConfigs = buildConfigurationsForPackages(services, providerName, environment, manualVcsRoot, configuration)
+
+    // Create build configs for sweepers
+    val preSweeperConfig = buildConfigurationForPreSweeper(sweepers, providerName, manualVcsRoot, configuration)
+    val postSweeperConfig = buildConfigurationForPostSweeper(sweepers, providerName, manualVcsRoot, configuration)
+
+    // Add trigger to last step of build chain (post-sweeper)
+    val triggerConfig = NightlyTriggerConfiguration(environment, branchRef)
+    postSweeperConfig.addTrigger(triggerConfig)
+    
     return Project{
 
-        // Create build configs for each package defined in packages.kt
-        var buildConfigs = buildConfigurationsForPackages(packages, providerName, "google<%= "-" + version unless version == 'ga' -%>", environment, manualVcsRoot, branchRef, configuration)
-        buildConfigs.forEach { buildConfiguration ->
+        // Register build configs in the project
+        buildType(preSweeperConfig)
+        (packageConfigs + servicePackageConfigs).forEach { buildConfiguration ->
             buildType(buildConfiguration)
         }
+        buildType(postSweeperConfig)
+
+        // Set up dependencies between builds using `sequential` block
+        // Acc test builds run in parallel
+        sequential {
+            buildType(preSweeperConfig)
+
+            parallel{
+                (packageConfigs + servicePackageConfigs).forEach { buildConfiguration ->
+                    buildType(buildConfiguration)
+                }
+            }
+
+            buildType(postSweeperConfig)
+        }
```

Comment:
This code is where all the build configurations are put in the Project being created, and the `sequential` and nested `parallel` block are what create dependencies between builds.